ui/app/AppLayouts/Wallet/views/HistoryView.qml

```diff
@@ -50,7 +50,7 @@ Item {
 
     StyledText {
         id: nonArchivalNodeError
-        visible: RootStore.isNonArchivalNode
+        visible: RootStore.isNonArchivalNode()
```

Comment:
This changes removed binding on `isNonArchivalNode` property and the signal will not update visibility of the component. Is it okay?

Reply:
RootStore declare a function, that is why I changed it to that, you mean that I should remove the parenthesis?

```
function isNonArchivalNode() {
        return walletModel.isNonArchivalNode
    }
```

Reply:
@borismelnik  NVM, I switch it to a property, should be ok now?

Reply:
```
  proc getIsNonArchivalNode(self: View): QVariant {.slot.} =
    return newQVariant(self.isNonArchivalNode)

  proc isNonArchivalNodeChanged(self: View) {.signal.}

  proc setIsNonArchivalNode*(self: View, isNonArchivalNode: bool) =
    self.isNonArchivalNode = isNonArchivalNode
    self.isNonArchivalNodeChanged()

  QtProperty[QVariant] isNonArchivalNode:
    read = getIsNonArchivalNode
    notify = isNonArchivalNodeChanged
```
It's a qt property declaration where `notify` means that the property provides reactive changes. But yes... looks like someone did it wrong before you... So, if you call `method setIsNonArchivalNode*[T](self: Module[T], isNonArchivalNode: bool) `, `visible` property in qml never change. in qml you need something like `visible: visible: RootStore.isNonArchivalNode`

Reply:
Yep, I did the change. Good catch